src/ui/button/button.tsx

```diff
@@ -20,6 +20,7 @@ export interface IButtonProps {
     | 'tertiary'
     | 'primary-dark'
     | 'diestruction'
+    | 'no-border'
```

Comment:
In my [PR](https://github.com/ForomePlatform/Anfisa-React-Client/pull/825) I made the same as a separate prop. But I'm not sure what's better

Reply:
I think a variant would be better that a prop, since a prop is applicable to each variant which may cause some unexpected issues in displaying.